modules/compute/gke-node-pool/main.tf

```diff
@@ -266,7 +266,7 @@ resource "google_project_iam_member" "node_service_account_artifact_registry" {
 
 resource "null_resource" "install_dependencies" {
   provisioner "local-exec" {
-    command = "pip3 install pyyaml argparse"
+    command = "pip3 install pyyaml argparse --break-system-packages"
```

Comment:
This is something I need on my cloudtop as of now.

Reply:
If it is something we need to check in, should we create a new top level variable to make sure we can turn it off in production ? 
This is another argument to have a separate set of terraform modules for the private repo. 
Should we discuss this with carson ?